index-navbar.html

```diff
@@ -0,0 +1,183 @@
+<!DOCTYPE html>
+<html lang="nl">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+    <!--fonts-->
+    <link href="https://fonts.googleapis.com/css?family=Open+Sans" rel="stylesheet">
+    <!--CSS stylesheets-->
+    <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/css/bootstrap.min.css" integrity="sha384-ggOyR0iXCbMQv3Xipma34MD+dH/1fQ784/j6cY/iJTQUOhcWr7x9JvoRxT2MZw1T" crossorigin="anonymous">
+    <link rel="stylesheet" href="stylesheets/../style.css">
+    <link href="https://cdnjs.cloudflare.com/ajax/libs/summernote/0.8.12/summernote-bs4.css" rel="stylesheet">
+
+    <!--JS-->
+    <script type="text/javascript" src="scripts.js"></script>
+    <script src="https://code.jquery.com/jquery-3.2.1.slim.min.js"></script>
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.11.0/umd/popper.min.js"></script>
+    <script src="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0-beta/js/bootstrap.min.js"></script>
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/summernote/0.8.12/summernote-bs4.js"></script>
+
+    <title>Showkorps Concento</title>
+</head>
+
+<body>
+
+    <div class="section-container">
+
+        <!--menu nav-->
+        <div class="section-navbar">
+            <nav class="topnav">
+                <div class="row">
+                    <div class="col-sm-4">
+                        <img class="logo" src="logo-concento.jpg" alt="logo concento" href="#logo">
+                    </div>
+                    <div class="col-sm-8">
+                        <ul class="menu-bar-right">
+                            <li class="menuAbout"><a href="#over-ons">Over Ons</a>
+                                <ul class="subAbout">
+                                    <li><a href="#geschiedenis">Geschiedenis</a></li>
+                                    <li><a href="#vlog">Vlog</a></li>
+                                    <li><a href="#foto&video">Foto & Video</a></li>
+                                </ul>
+                            </li>
+                            <li><a href="#nieuws">Nieuws</a></li>
+                            <li><a href="#agenda">Agenda</a></li>
+                            <li><a href="#boeking-form" id="boeking">Boeking</a></li>
+                            <li><a href="#lidworden">Lid Worden</a></li>
+                            <li class="menuExtra"><a href="#extra">Extra</a>

```

Comment:
Be consistent. You have classes that are camel cases and other have `-` as separator. 

I would call it menu-extra

Reply:
Classes are renamed now with a separator.